dvc/command/checkout.py

```diff
@@ -26,6 +85,18 @@ def add_parser(subparsers, parent_parser):
         help=CHECKOUT_HELP,
         formatter_class=argparse.RawDescriptionHelpFormatter,
     )
+    checkout_parser.add_argument(
+        "--show-changes",
+        action="store_true",
+        default=False,
+        help="Show stats of changes",
+    )
+    checkout_parser.add_argument(
+        "--summary",
+        action="store_true",
+        default=False,
+        help="Show summary of changes",
+    )
```

Comment:
Not sure if we need these as options, seems like we could show both by-default.

Reply:
Good point! We could make it default, but, I'll wait for product feedback.

Reply:
Would love to see them by default (summary at least)! 

may be --show-json to being able to script this, but it's another PR probably

Reply:
If summary is default, then we can name option -l (--long) or something like this to show all of the output

Reply:
Like we discussed below around `git checkout`, it is never silent. It'll display every changes if there's any unless silenced by `-q`.

Okay, the api already gives us a dict of checkouts. So, it should be easy to implement. But, let's keep it in another PR.

Reply:
This is now the default (if there's no changes, there's no output though).